lib/tests/streamlit/write_test.py

```diff
@@ -420,15 +419,6 @@ def test_unknown_arguments(self):
             in logs.records[0].msg
         )
 
-    def test_spinner(self):
-        """Test st.spinner."""
-        # TODO(armando): Test that the message is actually passed to
-        # message.warning
-        with patch("streamlit.delta_generator.DeltaGenerator.empty") as e:
-            with st.spinner("some message"):
-                time.sleep(0.15)
-            e.assert_called_once_with()
```

Comment:
This test doesn't make any sense to be in the `write_test` since it has nothing to do with `st.write`.